### ISSUE TITLE
Plug-in page

### ISSUE THREAD

**Issue by chiodicg** [cypress-io/cypress-documentation#3750]
## I'm submitting a...

```
[x] Bug report
[ ] Content update
[ ] Process update (build, deployment, ... )
```

## Type of bug / changes

The plug-in table at https://docs.cypress.io/plugins/directory needs to be formatted for small screens.
The titles are sometimes overlapping and the text goes out of the table edges. I am attaching a screenshot from my mobile. 
![59535A30-EA84-49EC-B6B3-D0302099BA5D](https://user-images.githubusercontent.com/71133849/112852667-0ce06180-90a4-11eb-8fcb-ef47667c99e2.jpeg)

**Comment by amirrustam** [comment]
Prioritized: DX-51

**Comment by matthamil** [comment]
Resolved by #3770
